### DISCUSSION TITLE
RACK: equipment occupying 1/2 U Vertically

### DISCUSSION THREAD

**Discussion by jprodriguesc1** [netbox-community/netbox#12306] [Ideas]
Hey guys.
I have some equipment that occupies 1/2 vertically, such as the Cisco ASA5506-X and the Khomp UMG FXS 240.
In my rack I place one next to the other, occupying the same U of the rack.
I can represent this in GLPI. I will leave an image.
![cisco](https://user-images.githubusercontent.com/70758953/233382963-123b0cd8-0545-4f6b-a06a-c69d741d0694.png)

In Netbox is it possible for me to leave it the same way?

**Comment by mtinberg** [comment]
If you look in the mailing list a few days ago Brian Candler answered a very similar question from someone else whether modeling the shelf as a parent Device with two Device Bays and the small units as child Devices or whether to just model the self as a 1U device and the devices on it as associated with the Rack but not in any particular unit (unracked devices).


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: João ***@***.***>
Sent: Thursday, April 20, 2023 8:35 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] RACK: equipment occupying 1/2 U Vertically (Discussion #12306)


Hey guys.
I have some equipment that occupies 1/2 vertically, such as the Cisco ASA5506-X and the Khomp UMG FXS 240.
In my rack I place one next to the other, occupying the same U of the rack.
I can represent this in GLPI. I will leave an image.
[cisco]<https://user-images.githubusercontent.com/70758953/233382963-123b0cd8-0545-4f6b-a06a-c69d741d0694.png>

In Netbox is it possible for me to leave it the same way?

**Reply by candlerb** [reply]
Also, you mean half of a 1U space *horizontally*.  Netbox now supports 1/2U vertical units.

**Reply by jprodriguesc1** [reply]
> Se você olhar na lista de discussão alguns dias atrás, Brian Candler respondeu a uma pergunta muito semelhante de outra pessoa, seja modelando a prateleira como um dispositivo pai com dois compartimentos de dispositivos e as unidades pequenas como dispositivos filhos ou apenas modelando a si mesmo como um 1U dispositivo e os dispositivos nele associados ao Rack, mas não em nenhuma unidade específica (dispositivos sem rack). — Mark Tinberg ***@***.***> Divisão de Tecnologia da Informação-Serviços de Rede Universidade de Wisconsin-Madison
> […](#)
> ________________________________ De: João ***@***.***> Enviado: quinta-feira, 20 de abril de 2023 8h35 Para: netbox-community/netbox ***@***.***> Cc: Inscrito ***@***.***> Assunto: [netbox-community/netbox] RACK: equipamento ocupando 1/2 U verticalmente (Discussão #12306 ) Olá pessoal. Tenho alguns equipamentos que ocupam 1/2 na vertical, como o Cisco ASA5506-X e o Khomp UMG FXS 240. No meu rack coloco um ao lado do outro, ocupando o mesmo U do rack. Eu posso representar isso no GLPI. vou deixar uma imagem. [cisco]< https://user-images.githubusercontent.com/70758953/233382963-123b0cd8-0545-4f6b-a06a-c69d741d0694.png > No Netbox é possível eu deixar do mesmo jeito? — Responda a este e-mail diretamente, visualize-o no GitHub< #12306 > ou cancele a inscrição<https://github.com/notifications/unsubscribe-auth/AAS7UM4NEYFI2SE7K7AD4DLXCE3TRANCNFSM6AAAAAAXFQRXEM >. Você está recebendo isto porque se inscreveu neste tópico.ID da mensagem: ***@***.***>

Hi mark,

Thank you very much for the answer, I temporarily made the equipment with a height of 0.5 (1/2 U), adding the 2 would occupy 1U.
I know it's a visual detail, but in another rack I have 2 Dell VRTX servers in a tower, and it occupies 10U.
As a provisional measure I placed it with the measure of 5U simulating it horizontally.
![vrtx netbox](https://user-images.githubusercontent.com/70758953/233449660-4b085cdb-9d05-4c58-a3b9-385de2aac68e.png)

These were the solutions I adopted until Netbox supported vertical units.

**Reply by jprodriguesc1** [reply]
> Also, you mean half of a 1U space _horizontally_. Netbox now supports 1/2U vertical units.


Hi Brian,

Many thanks for the reply.
Got it, it would be really nice to add this function in a new update.
netbox is an excellent tool!!
Thanks

**Reply by candlerb** [reply]
Just to confirm, that's two 5U chassis, each one has four blade slots?

In that case, you should create a "parent device" which takes up 5U, and has four "device bays".  Then the blades themselves are "child devices" which fit into those bays. Netbox models this quite nicely.

**Reply by jprodriguesc1** [reply]
> Just to confirm, that's two 5U chassis, each one has four blade slots?
> 
> In that case, you should create a "parent device" which takes up 5U, and has four "device bays". Then the blades themselves are "child devices" which fit into those bays. Netbox models this quite nicely.

Brian,
Dell's VRTX server comes in two versions: a rack version and a tower version.
And I have both versions in our datacenter.
This image refers to 2 tower servers that are together side by side.
To represent them on the rack I simulated as if they were "lying" horizontally, to solve the problem.

About the child devices had already started to create. And exactly as you mentioned, each device has 4 blades, and these blades have Xenserve installed, virtualizing some services.
As soon as I finish the physical part I will dedicate myself to the virtualization part.

**Reply by candlerb** [reply]
> This image refers to 2 tower servers that are together side by side.

Ah right - so the image is what's in Netbox, rather than reality. I get it now. It's a bit hard to find a photo being end-of-life, but [here is it](https://www.dell.com/en-uk/shop/cty/pdp/spd/poweredge-vrtx).

I think your solution is as good as you can get with Netbox today. I don't think it's possible for a single device to be both "parent" and "child" (so you can't have shelf -> tower -> blade).  The only other option I can think of is a 10U device which represents "a pair of VRTX towers sitting on a shelf"

Essentially, your Netbox shows what you would have *preferred* to have, which is two 5U rackmount servers :-)

**Reply by deeconwastaken** [reply]
@jprodriguesc1 Can you please provide an device type export for the Dell VRTX ? Or maybe even upload them to the Device Type Library? Thank you.
